Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -702,6 +702,20 @@ def inclusion {x y : Subgraph G} (h : x ≤ y) : x.coe →g y.coe where
   toFun v := ⟨↑v, And.left h v.property⟩
   map_rel' hvw := h.2 hvw
 
+@[simp]
+lemma inclusion_edge_apply_coe {x y : Subgraph G} (h : x ≤ y) (e : Sym2 x.verts) :
+    Sym2.map Subtype.val (Sym2.map (inclusion h) e) = Sym2.map Subtype.val e := by
```

Comment:
A more natural phrasing for this one would be about the composition of Subtype.val with inclusion, without mention of Sym2. Then this one and the latter should follow easily. Would you mind replacing with that?